pkg/host-timesync-daemon/main.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/linuxkit/virtsock/pkg/vsock"
```

Comment:
Yes, it's alpha in each "paragraph". Most projects seem to separate stdlib ones like `fmt` etc from third party ones with a blank line. Some block them all together, others group them by functional area etc.